lib/Command/ExpireGroupVersions.php

```diff
@@ -22,23 +22,38 @@
 namespace OCA\GroupFolders\Command;
 
 use OC\Core\Command\Base;
+use OCA\Files_Trashbin\Expiration;
 use OCA\Files_Versions\Versions\IVersion;
+use OCA\GroupFolders\Trash\TrashBackend;
 use OCA\GroupFolders\Versions\GroupVersionsExpireManager;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
 class ExpireGroupVersions extends Base {
+	/** @var GroupVersionsExpireManager */
 	private $expireManager;
 
-	public function __construct(GroupVersionsExpireManager $expireManager) {
+	/** @var TrashBackend */
+	private $trashBackend;
+
+	/** @var Expiration */
+	private $expiration;
+
+	public function __construct(
+		GroupVersionsExpireManager $expireManager,
+		TrashBackend $trashBackend,
+		Expiration $expiration
+	) {
 		parent::__construct();
 		$this->expireManager = $expireManager;
+		$this->trashBackend = $trashBackend;
+		$this->expiration = $expiration;
 	}
 
 	protected function configure() {
 		$this
 			->setName('groupfolders:expire')
-			->setDescription('Trigger expiry of versions for files stored in group folders');
+			->setDescription('Trigger expiry of versions and trashbin for files stored in group folders');
```

Comment:
If we really mean to change the behavior of the expire command we need to update the README at least.
Also, why is the commit labeled "*Temporarily* add to the expire command"?

Reply:
Probably because we still need to make sure that the parameter in https://docs.nextcloud.com/server/latest/admin_manual/configuration_files/trashbin_configuration.html#background-job are taken in count. Should be easy to do

Reply:
Ok done :) And added a todo for doc

Reply:
> Probably because we still need to make sure that the parameter in https://docs.nextcloud.com/server/latest/admin_manual/configuration_files/trashbin_configuration.html#background-job are taken in count.

Hm, the `background_job_expire_trash` is only for the background job, but the commit is about the occ `expire` command, no? Or am I misreading the code?

Anyway, with the docs mentioning that the `expire` command triggers expiration of versions **and** trashbin, this should be ok.